Makefile

```diff
@@ -256,5 +260,5 @@ bin/actionlint-$(ACTIONLINT_VERSION): | $(BIN_DIR)
 	@mv bin/actionlint $@
 
 .PHONY: actionlint-lint
```

Comment:
need to update this .PHONY to match the change below

Reply:
Nice catch! Fixed.